samples/MegaDrone/CMakeLists.txt

```diff
@@ -27,7 +27,8 @@ target_link_libraries( megadrone log oboe )
 # Make the Oboe public headers available to our app
 include_directories (${OBOE_DIR}/include)
 
-# Append compiler optimization flags to both the debug and release build variants
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Ofast")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Ofast")
-
+# Enable optimization for release build; it could be enabled for debug build
+# if performance is of interest.
+target_compile_options(megadrone
+    PRIVATE
+        -std=c++14 -Wall $<$<CONFIG:RELEASE>:-Ofast>)
```

Comment:
Why add `-std=c++14`? I don't think the app needs any C++14 features in which case it makes sense to leave this out.

Reply:
Also, for this particular example `-Ofast` should be enabled for both debug and release because it's a performance related sample and we don't want first-time developers thinking there's a bug in the code because of audio glitches in the default (debug) configuration.

Reply:
I thought we have email said that the standard is to use C++14 as a standard configuration, and enable -Ofast for release.

Reply:
[clang -g doc](https://clang.llvm.org/docs/CommandGuide/clang.html#code-generation-options) points out that -g  works best with -o0.  Implying -ofast enabled, source level debugging may have trouble.  I think to enable it for release; for debugging version, leave it out and leave users to add it.  thanks

Reply:
I agree that `-O0` should be used in normal circumstances but for the MegaDrone sample it will almost certainly glitch and hence should be left as `-Ofast` for all variants.

Reply:
done with next checkin, thx